nemo/collections/asr/data/vocabs.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Why create a new vocab class rather than using what you created with ASR's parsers?

Reply:
It's not flexible enough and lack of explicit phonemes support.
We may want to merge them now or someday. The only problem is: universal support vocabularies & parsers is a real hard task. We need:
* Chars/Phonemes
* Possibly Different Languages
* Normalizing
* Encoding/Decoding
* Support & reserve all possible special chars
* Support updating set of graphemes to use (fine-tunning)
* Maybe some other usecases

Reply:
I don't want to implement this logic in TalkNet PR :)

Reply:
Honestly, I don't have time for this in a nearest month. Supporting all possible vocab use cases (for every model, for every team) would requires a lot of work, so I can't give you my 100% commitment.
However, in general, I am very interested in building well-engineered multi-purpose parsers.
Sounds like a good task for other day.